code/src/components/ListedMovie.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// Styling
+import 'assets/MovieList.css';
+
+// ----------------------------------------------------------------------------------------
+
+const ListedMovie = ({ id, title, releaseDate, posterImgSrc }) => {
+  return (
+    <Link key={id} to={`/movies/${id}`} className="movie-list--movie-wrapper">
```

Comment:
Maybe split it into 
<Link
 key={id}
 to={blabla}
/>

Reply:
Moreover, we don't need `key` attribute as we are not performing any loop